package.json

```diff
@@ -2,8 +2,7 @@
   "name": "solid-web-app-template",
   "version": "0.0.4",
   "description": "A template for a Solid Web app.",
-  "main": "index.js",
-  "type": "module",
```

Comment:
Why was the package (and related code) changed to CommonJS?

Reply:
It wasn't necessary to use ES6 modules, it can however be changed back without any issues.

Reply:
The ideal is to change the last amount of code possible to get it working with React. So if it's not needed to switch to CommonJS then we should not do that.

Reply:
Didn't know it was heavier :laughing: changed it back to ES modules in [commit 716eaa1](https://github.com/SolidLabResearch/solid-web-app-react-template/pull/1/commits/716eaa133e209e3a0785953303f5fa910dce4815)